adapter/kratos/log.go

```diff
@@ -0,0 +1,64 @@
+package logs
+
+import (
+	coopLog "github.com/coopnorge/go-logger"
```

Comment:
FYI added an alias because linter could not resolve correct reference to package linter error was: `undefined: logger (typecheck)`